pkg/deviceshifu/deviceshifumqtt/deviceshifumqtt.go

```diff
@@ -72,15 +76,20 @@ func New(deviceShifuMetadata *deviceshifubase.DeviceShifuMetaData) (*DeviceShifu
 			if token := client.Connect(); token.Wait() && token.Error() != nil {
 				panic(token.Error())
 			}
-			MQTTTopic = *mqttSetting.MQTTTopic
-			sub(client, MQTTTopic)
 
-			HandlerMetaData := &HandlerMetaData{
-				base.EdgeDevice.Spec,
-			}
+			for instruction, properties := range mqttInstructions.Instructions {
+				MQTTTopic = properties.MQTTInstructionProperty.MQTTTopic
+				sub(client, MQTTTopic)
 
-			handler := DeviceCommandHandlerMQTT{HandlerMetaData}
-			mux.HandleFunc("/"+MqttDataEndpoint, handler.commandHandleFunc())
+				HandlerMetaData := &HandlerMetaData{
+					base.EdgeDevice.Spec,
+					instruction,
+					properties.MQTTInstructionProperty,
+				}
+
+				handler := DeviceCommandHandlerMQTT{HandlerMetaData}
```

Comment:
shall we do a bench mark test for this feature?
eg, how many topics one mqttdeviceshifu can hold?
how many messages/second can send to these topics? 
meanwhile watch the cpu/memory status?